packages/core/src/di/interface/BUILD.bazel

```
@@ -0,0 +1,18 @@
+package(default_visibility = ["//visibility:public"])
```

this doesn't seem right. we don't want the visibility to be public. use the following instead:
```suggestion
package(default_visibility = ["//packages/core:__subpackages__"])
```

Please refactor all the internal build files to use this restricted visibility.

---

fixed